android/app-example/src/androidTest/java/com/badoo/ribs/example/rib/dialog_example/DialogExampleTest.kt

```diff
@@ -2,7 +2,7 @@ package com.badoo.ribs.example.rib.dialog_example
 
 import com.badoo.common.ribs.RibsRule
 import com.badoo.ribs.RibTestActivity
-import com.badoo.ribs.core.directory.Directory
+import com.badoo.ribs.customisation.RibCustomisationDirectory
```

Comment:
We will have to migrate all written RIBs to new naming.
Is it fine?

Reply:
Yes.